### PR TITLE
Credits page build

### PR DESCRIPTION
**Pull request by vishisht-dubey**

### CODE REVIEW

**Comment by vishisht-dubey** [comment]
Hiii @dev-raj-1729 Please review this pr

File: app/credits/page.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"use client";
+import React from "react";
+import { Typography } from "@material-tailwind/react";
+import Team from "../../components/creditcomponents/TeamComponent";
+import { CreditsDataType } from "../../components/creditcomponents/TeamComponent";
+import { dummyData } from "../../configs/creditPeople";
```
Comment (Shubhrajyoti-Dey-FrosTiK) [inline app/credits/page.tsx]: Change the name from `dummyData` to `teamMembers`

File: app/credits/page.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+              linkedinURL={value.linkedinURL}
+              githubURL={value.githubURL}
+              mailID={value.mailID}
+              key={key}
+            />
+          ))}
+      </div>
+      <div>
+        {/* <Footer /> */}
+      </div>
```
Comment (Shubhrajyoti-Dey-FrosTiK) [inline app/credits/page.tsx]: Remove comment and `div`

File: app/layout.tsx [hunk 3]
```diff
@@ -41,11 +41,13 @@ export default function RootLayout({
                 <MobileDrawer ref={targetRef} schema={DemoNavSchema}>
                   <div>{children}</div>
                 </MobileDrawer>
+                <Footer/>
               </div>
               <div className=" hidden sm:block">
                 <NavbarNested ref={targetRef} schema={DemoNavSchema}>
                   {children}
                 </NavbarNested>
+                <Footer/>
```
Comment (Shubhrajyoti-Dey-FrosTiK) [inline app/layout.tsx]: Add `Footer` to the end rather than declaring it 2 times

File: app/page.tsx [hunk 4]
```diff
@@ -15,6 +15,7 @@ import { selectIdStore } from "../store/states/idStore";
 import { useEffect, useState } from "react";
 import axios from "axios";
 import { IconEdit } from "@tabler/icons";
+import Footer from "../components/footer/Footer";
```
Comment (Shubhrajyoti-Dey-FrosTiK) [inline app/page.tsx]: Why ??

File: app/register/recruiter/page.tsx [hunk 5]
```diff
@@ -9,7 +9,6 @@ import RecruiterAndCompanyRegisterSchema from "../../../configs/CompanyRegisterS
 import { handleSignUpWithEmailPassword } from "../../../firebase/auth";
 import useExportableFormData from "../../../hooks/useExportableFormData";
 import { selectForm } from "../../../store/states/formSlice";
-
```
Comment (Shubhrajyoti-Dey-FrosTiK) [inline app/register/recruiter/page.tsx:12]: Remove the comment
And why remove line?

**Review by Shubhrajyoti-Dey-FrosTiK — CHANGES_REQUESTED** [review]

File: app/credits/page.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+export default function Credits() {
+  return (
+    <div className="border-1 border-purple-500 m-5 grid justify-center">
+      <div className="mb-4">
+        <Typography variant="h1">Team Frostik</Typography>
+      </div>
+      <div className="grid lg:grid-cols-3 md:grid-cols-2 sm:grid-cols-1 gap-6">
+        {teamMembers &&
+          teamMembers.map((value: CreditsDataType, key: any) => (
```
Comment (Shubhrajyoti-Dey-FrosTiK) [inline app/credits/page.tsx:16]: Change `value` to `member`

**Review by Shubhrajyoti-Dey-FrosTiK — CHANGES_REQUESTED** [review]

**Pull request merged**